x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -405,7 +408,28 @@ export const SymbolDefinitions = styled(SymbolDefinitionsComponent)`
   height: 0;
 `;
 
-export const useResolverTheme = (): { colorMap: ColorMap; nodeAssets: NodeStyleMap } => {
+const processTypeToCube: Record<ResolverProcessType, keyof NodeStyleMap> = {
+  processCreated: 'runningProcessCube',
+  processRan: 'runningProcessCube',
+  processTerminated: 'terminatedProcessCube',
+  unknownProcessEvent: 'runningProcessCube',
+  processCausedAlert: 'runningTriggerCube',
+  unknownEvent: 'runningProcessCube',
+};
+
+export function nodeType(processEvent: ResolverEvent): keyof NodeStyleMap {
```

Comment:
can you comment this.

Reply:
https://github.com/elastic/kibana/pull/69098/commits/318eba28676db62960866e5fccb4558f918a5fb0